web/client-api/src/test/java/io/deephaven/web/client/api/ColumnStatisticsTestGwt.java

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.web.client.api;
+
+import elemental2.core.JsMap;
+import elemental2.promise.Promise;
+
+public class ColumnStatisticsTestGwt extends AbstractAsyncGwtTestCase {
+    private final TableSourceBuilder tables = new TableSourceBuilder()
+            .script("from deephaven import empty_table")
+            .script("staticTable", "empty_table(3).update([\"A = i % 2\", \"B = `` + A\"])");
+
+    public void testStatsWithUniqueCounts() {
+        connect(tables)
+                .then(table("staticTable"))
+                .then(table -> Promise.all(
+                        table.getColumnStatistics(table.findColumn("A"))
+                                .then(stats -> {
+                                    JsMap<String, Double> uniqueValues = stats.getUniqueValues();
+                                    assertEquals(0, uniqueValues.size);
```

Comment:
Are unique values only calculated for certain column types?

Reply:
Correct, as with DHE, we only find uniques for non-numeric, non-datetime, types.